express-forms/src/index.js

```diff
@@ -1,17 +1,31 @@
 const express = require('express');
+const bodyParser = require('body-parser');
 
 const app = express();
 
-app.get('/', (req, res) => {
-  return res.redirect('/form-with-get');
-});
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.set('view engine', 'ejs');
 
-app.get('/form-with-get', (req, res) => {
-  return res.render('form-with-get');
-});
 
-app.get('/form-with-post', (req, res) => {
-  return res.render('form-with-post');
+app.get('/', (req, res) => (
+  res.redirect('/form-with-get')
+));
+
+app.get('/form-with-get', (req, res) => (
+  res.render('form-with-get')
+));
+
+app.get('/form-with-post', (req, res) => (
+  res.render('form-with-post')
+));
+
+app.get('/submit-form-with-get', (req, res) => (
+  res.send(req.query)
+));
+
+app.post('/submit-form-with-post', (req, res) => {
+  res.send(req.body);
```

Comment:
Should you send full `body` or `query` object back in response?
I think it is always good to process `request` and respond with required information.